### ISSUE TITLE
ATM6 1.3.4c Crashes after applying Soartex Fanver Resource Pack

### ISSUE THREAD

**Issue by NetMasterFlex** [AllTheMods/ATM-6#571]
**Game Crashes prior to joining world.  Happens after applying resource pack to the modpack.

**To Reproduce**
1. Go to 'Options'
2. Click on 'Resource Packs'
3. Scroll down to 'Select Resource Pack (Manually added outside of game) Soartex_Fanver_Universal_1.16.x_2020_11_27.zip
4. Click and move to the right side of the screen 
5. Click Done.

Wait for the Resource Pack to attempt applying to the pack.

During the last part of stitching the pack, the game crashes back to launcher.

Note: Using Multi-MC to launch the pack.  10GB RAM dedicated  i7 Quad Core CPU, Macbook Pro Early 2015.

**Expected behavior**
Tesxture pack applies and I can assume server selection and join a world.

**Additional context**
Attaching some Crash Reports.
[crash-2020-12-15_18.54.48-client.txt](https://github.com/AllTheMods/ATM-6/files/5700510/crash-2020-12-15_18.54.48-client.txt)
[crash-2020-12-15_20.20.55-fml.txt](https://github.com/AllTheMods/ATM-6/files/5700511/crash-2020-12-15_20.20.55-fml.txt)

**Comment by gbusrt** [comment]
i had the same thing happen when i tried purebdcraft 64 and 128 and with faithfull
sadly texture packs are not high on the bug fix list , better to fix game that looks

**Comment by DeathOfTime** [comment]
have you tried adding the resource pack directly to options.txt. i've heard exiting the game directly after pushing done works as well

**Comment by gbusrt** [comment]
No not tried that were is the options.txt file and what must be added ?

**Comment by NetMasterFlex** [comment]
Yeah please provide detail about the options text file. I believe I’ve done that in the past will little success, but will try again given clarity on what to edit and where in options text file. Please provide an example if you can.  I will also try exiting the game after clicking done. Haven’t tried that. Will report back. 

Thanks

NMF 

PS. Works fine on 1.3.3. No workarounds needed.

**Comment by NetMasterFlex** [comment]
UPDATE!

Ok so I tried the method of manually entering in the texterpack as suggested, but when the game menu loaded, I could tell right away that the texture pack wasn't applied.  The bacground of the menu is that of dark smoothy cobblestone.  I clicked Options > Resource Packs.  Saw that the resource pack was on the left side.  I added both Quark Programmer TP and Soartex Fanver TP to the right hand side.  Clicked Done, then waited for the opportunity to X out of the game.  Initially the screen freezes (Beach Ball for me, Mac user), then I X'd out.  Upon reload the Resource packs were indeed applied.  My server wasn't up so I loaded a single player game and it loaded without crashing and textures were present.

Recap Workaround as suggested by @DeathOfTime:

"i've heard exiting the game directly after pushing done works as well"

After clicking DONE, exit the game when able to do so.  Do not End Task (Windows) or Force Quit (Mac).  Wait for an opportunity to exit the game, then reload the game and it will hopefully work as it did for me.

**Comment by NetMasterFlex** [comment]
UPDATE!

Ok so I tried the method of manually entering in the texterpack as suggested, but when the game menu loaded, I could tell right away that the texture pack wasn't applied.  The bacground of the menu is that of dark smoothy cobblestone.  I clicked Options > Resource Packs.  Saw that the resource pack was on the left side.  I added both Quark Programmer TP and Soartex Fanver TP to the right hand side.  Clicked Done, then waited for the opportunity to X out of the game.  Initially the screen freezes (Beach Ball for me, Mac user), then I X'd out.  Upon reload the Resource packs were indeed applied.  My server wasn't up so I loaded a single player game and it loaded without crashing and textures were present.

Recap Workaround as suggested by @DeathOfTime:

"i've heard exiting the game directly after pushing done works as well"

After clicking DONE, exit the game when able to do so.  Do not End Task (Windows) or Force Quit (Mac).  Wait for an opportunity to exit the game, then reload the game and it will hopefully work as it did for me.
